src/engine/GraphStoreProtocol.h

```diff
@@ -64,9 +57,26 @@ class GraphStoreProtocol {
       const std::string_view& mediatype);
 
   // Throws the error if an HTTP method is not supported.
-  [[noreturn]] static void throwUnsupportedHTTPMethod(
+  [[noreturn]] static void throwNotYetImplementedHTTPMethod(
       const std::string_view& method);
 
+  // Aborts the request with an HTTP 204 No Content if the request body is
+  // empty.
+  static void throwIfRequestBodyEmpty(const auto& request) {
+    if (request.body().empty()) {
+      throw HttpError(boost::beast::http::status::no_content,
+                      "Request body is empty.");
+    }
+  }
+
+  static std::string truncatedStringRepresentation(std::string type,
+                                                   const auto& request) {
+    // Graph store protocol requests might have a very large body. Limit
+    // the length used for the string representation.
+    return absl::StrCat("Graph Store ", type, " Operation\n",
+                        ad_utility::truncateOperationString(request.body()));
+  }
+
```

Comment:
Add a comment, and maybe add it to the other small cleanups pr, because it was in the preexisting code already.